src/frontend/index.html

```diff
@@ -5,106 +5,43 @@
 
   <body>
-    <div class="main-grid">
-      <!-- NAVIGATION -->
-      <div class="navigation-adaptive">
-        <!-- LOGO -->
-        <img id="main-logo" src="./img/logo.svg" alt="Telescope logo" />
-
-        <!-- NAV LINKS -->
-
-        <nav>
-          <a href="/login">
-            <button class="nav-button" class="login">
-              Login
-            </button>
-          </a>
-
-          <a href="#">
-            <button class="nav-button">
-              Feed
-            </button>
-          </a>
-
-          <a href="#">
-            <button id="participants" class="nav-button modal-button" data-modal="participants">
-              Participants
-            </button>
-          </a>
-
-          <a href="#">
-            <button class="nav-button">
-              Subscribe
-            </button>
-          </a>
-          <!-- SEARCH CONTAINER -->
-          <div class="search_container">
-            <div class="wrap">
-              <div class="search">
-                <input type="text" class="searchTerm" placeholder="Search" />
-                <button type="submit" class="searchButton">
-                  <i class="fa fa-search"></i>
-                </button>
-              </div>
-            </div>
-          </div>
-
-          <a>
-            <div id="google_translate_element2"></div>
-          </a>
-
-          <i id="mobile-burger-close" class="fas fa-times"></i>
-        </nav>
-
-        <!-- MOBILE HAMBURGER MENU (DEFAULT: HIDDEN) -->
-        <i id="mobile-burger" class="fas fa-bars"></i>
-      </div>
-
-      <!-- CONTENT CONTAINER-->
-      <div class="content">
-        <!-- Dynamically loaded content goes here -->
-      </div>
-
-      <!-- Structure for modal, hidden by default -->
-      <div class="modal-container">
-        <div class="modal-panel">
-          <h1>Modal</h1>
-          <section></section>
-        </div>
-      </div>
-    </div>
-
-    <script src="/scripts/main.js"></script>
-
-    <!-- Google translate Javascript -->
-    <script type="text/javascript">
-      function googleTranslateElementInit2() {
-        new google.translate.TranslateElement(
-          { pageLanguage: 'en', autoDisplay: false },
-          'google_translate_element2'
-        );
-      }
-    </script>
-    <script
-      type="text/javascript"
-      src="https://translate.google.com/translate_a/element.js?cb=googleTranslateElementInit2"
-    ></script>
+    <nav>
+      <a href="/auth/login">
+        <button class="nav-button" class="login">
+          Login
+        </button>
+      </a>
+
+      <a href="/admin/queues">
+        <button class="nav-button" class="login">
+          Admin Queues (Requires Authentication)
+        </button>
+      </a>
+
+      <a href="/auth/logout">
+        <button class="nav-button" class="login">
+          Logout
+        </button>
+      </a>
+    </nav>
+
+    <h1>Telescope</h1>
+
+    <p>NOTE: this page is going away soon, just for testing login/logout.</p>
```

Comment:
Not sure what this is referring too?

Reply:
This entire web page will be replaced by our GatsbyJS app.  I just have this here now for testing that the authenticated routes work.  I added this note so people didn't wonder why I had this page.